activejob/lib/active_job/test_helper.rb

```diff
@@ -77,14 +77,23 @@ def queue_adapter_for_test
     #       HelloJob.perform_later('jeremy')
     #     end
     #   end
-    def assert_enqueued_jobs(number, only: nil)
+    #
+    # If a queue name is passed, the number of times jobs enqueued to that specific queue can be asserted.
```

Comment:
How about `The number of times a job is enqueued to a specific queue can also be asserted`
Comments would then read:

```
    # Asserts that the number of enqueued jobs matches the given number.
    # ...
    #
    # If a block is passed, that block should cause the specified number of
    # jobs to be enqueued.
    #
    # ...
    #
    # The number of times a specific job is enqueued can be asserted.
    #
    # ...
    #
    # The number of times a job is enqueued to a specific queue can also be asserted.
    #
    # ...
```